packages/block-library/src/latest-comments/edit.js

```diff
@@ -31,8 +32,12 @@ const MIN_COMMENTS = 1;
 const MAX_COMMENTS = 100;
 
 export default function LatestComments( { attributes, setAttributes } ) {
-	const { commentsToShow, displayAvatar, displayDate, displayExcerpt } =
-		attributes;
+	// Handle backward compatibility in the editor
+	const displayContent =
+		attributes.displayContent ||
+		( attributes.displayExcerpt === false ? 'none' : 'excerpt' );
+
+	const { commentsToShow, displayAvatar, displayDate } = attributes;
```

Comment:
```suggestion
	const { commentsToShow, displayAvatar, displayDate, displayContent } =
		attributes;
```

At this point, the migration should have already been executed, so backward compatibility code is not necessary.